Source/CesiumRuntime/Public/Cesium3DTileset.h

```diff
@@ -132,6 +132,9 @@ class CESIUMRUNTIME_API ACesium3DTileset : public AActor,
       meta = (ClampMin = 0))
   int32 MaximumSimultaneousTileLoads = 20;
 
+  UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "Cesium|Tile Loading")
+  int32 maximumCachedBytes = 256;
```

Comment:
This should be upper case, consistent with the Unreal Engine coding standards. Also, it should be an int64, not an int32. And please copy/paste the doc comments from cesium-native to here as well so that they'll show up as a tooltip in Unreal Engine.

Reply:
Done !